contrib/opencensus-ext-azure/opencensus/ext/azure/common/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '1.2.dev0'
+__version__ = '1.1.8'
```

Comment:
Not 1.2.0?

Reply:
We could but we have been releasing patch versions since the changes have not been huge.